src/swap/redux/actions.js

```diff
@@ -33,3 +34,32 @@ export const submitEthWrapperAuthorize = () =>
     expiry: INFINITE_EXPIRY,
     delegate: WRAPPER_CONTRACT_ADDRESS,
   })
+
+export const trackSwapAllContracts = ({
```

Comment:
How does this track swap/cancels through the different swap contracts?

Reply:
Most tracking uses the event hash (keccak hash of the method signature) + contract address to filter for events.

This function *only* uses the event hash so it will find *all* events across *all* contracts that matches the method signature of the `swap` function. So the key here is the omission of the swap contract address, as is seen here in the old method: https://github.com/airswap/AirSwap.js/blob/develop/src/swap/redux/eventTrackingActions.js#L7

Reply:
Ahh I see, that makes sense. Thanks